packages/serverless-workflow-language-service/tests/getFileLanguage.test.ts

```diff
@@ -14,17 +14,15 @@
  * limitations under the License.
  */
 
-import { getFileLanguage, FileLanguage } from "@kie-tools/serverless-workflow-language-service/dist/editor";
+import {
+  FileLanguage,
+  getFileLanguage,
+  getFileLanguageOrThrow,
+} from "@kie-tools/serverless-workflow-language-service/dist/api";
 
 describe("getFileLanguage", () => {
   it("Checking file language with not valid inputs", () => {
-    // @ts-ignore
-    expect(getFileLanguage(null)).toBe(null);
-    // @ts-ignore
-    expect(getFileLanguage(undefined)).toBe(null);
```

Comment:
Removed as the type system already guarantees that it's not possible to use these values.

Reply:
I do this kind of checks because TypeScript doesn't check on runtime and I wanted to ensure that a wrong value doesn't break the system.